LifeCycleAssessment_Engine/Compute/EvaluateTM65.cs

```diff
@@ -0,0 +1,150 @@
+using BH.Engine.Matter;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Returns reinforcement volume.")]
+        [Input("name", "Name of the equipment type being evaluated. This name will be used to identify the complexity multiplier.")]
+        [Input("weight", "Weight of the equipment in kg.")]
+        [Input("quantity", "The amount of determined QuantityType.")]
+        [Input("materialComposition", "The material composition of the object being evaluated. This is a breakdown of materials by ratio.")]
+        [Output("environmentalMetric", "Percentage reinforcement within concrete building element m^3.")]
+        public static EnvironmentalMetric EvaluateTM65 (string name, double weight, double quantity, QuantityType quantityType, MaterialComposition materialComposition) //add optional refrigerant input after prototype is working.
+        {
+            // TM65 calculation 
+            // TotalWeight x ExplicitBulkByMass x 1.1 (ReplacementWeightIncrease) x Scale Up Factor x 1.3 (Buffer Factor) + RefrigerantTotalFromImageBelow
+
+            // Force input to lower case
+            name = name.ToLower();
+
+            double replacementWeightIncrease = 1.1; // is this constant or does this need to be variable? 
+            double bufferFactor = 1.3; // is this constant or does this need to be variable?
+            // Refrigerant to be added later
+            
+            // Define complexity lists
+            EPDComplexity scaleUpFactor = EPDComplexity.Undefined;
+
+            // Class 1 objects > scaleUpFactor == 1.1
+            List<string> class1 = new List<string>() { 
+                "pipe",
+                "cables",
+                "ducts",
+                "valves",
+                "fire alarm",
+                "access control",
+                "cable containment",
+                "electrical outlets",
+                "busbars"
+            };
+
+            // Class 2 objects > scaleUpFactor == 1.2
+            List<string> class2 = new List<string>() {
+                "pumps",
+                "luminaires",
+                "radiators",
+                "control panels",
+                "lighting control devices",
+                "sensors",
+                "thermal store"
+            };
+
+            // Class 3 objects > scaleUpFactor == 1.3
+            List<string> class3 = new List<string>() {
+                "air handling units",
+                "heat pumps",
+                "boilers",
+                "heat interface units",
+                "chillers",
+                "generator",
+                "MVHR",
+                "switchgear",
+                "UPS"
+            };
+
+            // Needs null handling
```

Comment:
Complexity to be explicit input, enum and method to have helper text on the input for this item giving examples of Class 1,2,3 complexities